include/juice/juice.h

```diff
@@ -64,6 +64,17 @@ typedef void (*juice_cb_gathering_done_t)(juice_agent_t *agent, void *user_ptr);
 typedef void (*juice_cb_recv_t)(juice_agent_t *agent, const char *data, size_t size,
                                 void *user_ptr);
 
+typedef struct juice_stun_binding {
+	const char *ufrag;
+	const char *pwd;
+
+	uint8_t family;
+	const char *address;
+	uint16_t port;
+} juice_stun_binding_t;
+
+typedef void (*juice_cb_stun_binding_t)(const juice_stun_binding_t *info);
```

Comment:
This should accept a `void *user_ptr` argument to allow passing some context along with the invocation.

See the declarations of `juice_cb_state_changed_t`, `juice_cb_candidate_t` etc above for an example.